rules/config

```diff
@@ -77,7 +77,7 @@ ENABLE_ORGANIZATION_EXTENSIONS = y
 #SONIC_PROFILING_ON = y
 
 # ENABLE_SYSTEM_TELEMETRY - build docker-sonic-telemetry for system telemetry support
-ENABLE_SYSTEM_TELEMETRY = y
+ENABLE_SYSTEM_TELEMETRY = n
```

Comment:
Why? because of size?

Reply:
barefoot 9.0.0 did not support telemetry and compilation was broken.